src/Symfony/Component/Cache/Tests/DependencyInjection/CachePoolPassTest.php

```diff
@@ -232,4 +233,35 @@ public function testChainAdapterPool()
         $this->assertInstanceOf(ChildDefinition::class, $doctrineCachePool);
         $this->assertSame('cache.app', $doctrineCachePool->getParent());
     }
+
+    public function testGlobalClearerAlias()
+    {
+        $container = new ContainerBuilder();
+        $container->setParameter('kernel.container_class', 'app');
+        $container->setParameter('kernel.project_dir', 'foo');
+
+        $container->register('cache.default_clearer', Psr6CacheClearer::class);
+
+        $container->setDefinition('cache.system_clearer', new ChildDefinition('cache.default_clearer'));
+
+        $container->setDefinition('cache.foo_bar_clearer', new ChildDefinition('cache.default_clearer'));
+        $container->setAlias('cache.global_clearer', 'cache.foo_bar_clearer');
+
+        $container->register('cache.adapter.array', ArrayAdapter::class)
+            ->setAbstract(true)
+            ->addTag('cache.pool');
+
+        $cachePool = new ChildDefinition('cache.adapter.array');
+        $cachePool->addTag('cache.pool', ['clearer' => 'cache.system_clearer']);
+        $container->setDefinition('app.cache_pool', $cachePool);
+
+        $this->cachePoolPass->process($container);
+
+        $definition = $container->getDefinition('cache.foo_bar_clearer');
+
+        $this->assertTrue($definition->hasTag('cache.pool.clearer'));
+        $this->assertCount(1, $arguments = $definition->getArguments());
+        $this->assertIsArray($firstArgument = reset($arguments));
+        $this->assertArrayHasKey('app.cache_pool', $firstArgument);
```

Comment:
Can we replace the three lines above with something like `$this->assertEquals(['app.cache_pool' => new Reference('app.cache_pool')], $definition->getArgument(0));`?

Reply:
Done